cookbooks/mantle_convection_with_continents_in_annulus/doc/mantle_convection_in_annulus.md

```diff
@@ -0,0 +1,89 @@
+existing rotational null space is removed by imposing no-net-rotation of
+the mantle. The boundaries have fixed temperatures of 3700K and 300K.
+Within the domain two compositional fields are defined: mantle and
+continent. The mantle domain consists of three different regions separated
+by the two major phase changes occurring at ~410- and ~660 km
+depths. We use the geodynamic World Builder {cite:t}`Fraters2019c` to
+set the initial temperature and compositional field distribution
+(see {numref}`fig:init_visc`, {numref}`fig:init_compositions` and  {numref}`fig:init_temperature`).
+
+
+```{figure-md} fig:init_visc
+<img src="viscosity.*" width="90%" />
+
+the initial viscosity field.
+```
+
+```{figure-md} fig:init_compositions
+<img src="continents.*" width="90%" />
+
+The initial position of the three continents.
+```
+
+```{figure-md} fig:init_temperature
+<img src="temperature.*" width="90%" />
+
+The initial temperature field.
+```
+
+Similar to {cite:t}`ulvrova:etal:2019`, we use viscous rafts as `continents' to aid with
+modelling one-sided subduction systems. We use three continents of
+5000, 5000 and 3000 km length, covering roughly 30% of the model
+surface. To avoid subduction of the continents, we assign the viscous
+rafts a reference density of 2916 kg/m3, which corresponds to a 400
+kg/m3 density contrast with the reference mantle density.
+Furthermore, to avoid deformation of the continents, we assign
+them a viscosity that equals the maximum cut-off viscosity in the model,
+i.e. 6e24 Pa s. We also configure three subduction zones
+in the initial set-up to seed the model with initial negative
+buoyancy.
+
+As the goal of the publication is to investigate
+how average slab sinking rates relate to the vigour of mantle convection and mixing,
+various rheologies are considered in the lower mantle, as
+show in {numref}`fig:visc-profiles`.
+We will focus on the 'R' (reference) profile in this cookbook (green line).
+
+```{figure-md} fig:visc-profiles
+<img src="viscosity_profiles.*" width="90%" />
+
+Radial viscosity profiles for the three models discussed in the original
+publication (solid lines)
+compared to viscosity profiles as used in other studies.
+See original publication for the references corresponding to the
+dashed lines.
+```
+
+After 500 Myr we see multiple subductions taking place as well as
+many plumes/upwellings, as shown in {numref}`fig:evolution`
+
+```{figure-md} fig:evolution
+<img src="evolution.*" width="90%" />
+
+Results of the reference model (R) after 500 Ma of mantle convection simulation. Snapshots of the model at t = 500 Ma are shown for the viscosity (a) and
+temperature (b) within the modelled domain, showing five slabs actively subducting below the continents (pink). c Solid lines indicate average surface velocity (blue)
+and mobility ratio M (Tackley, 2000) (red) of model R as well as their dashed time-averages Vsurf=2.1 cm/a and M = 2.218. d Radial velocity of all tracers defined as
```

Comment:
Perhaps a short phrase describing what the mobility ratio represents compared to the surface velocity (v_rms (surf)/ v_rms (whole mantle)) in case someone wondered what that specific expression was and looked it up in Tackley 2000 (should that be a bib citation?).